src/test/java/com/brew/oauth20/server/fixture/ClientModelFixture.java

```diff
@@ -13,7 +13,7 @@
 public class ClientModelFixture extends Fixture<ClientModel> {
```

Comment:
ModelFixture'lerimiz olmalı mı emin değilim. Zaten entity fixturelerimiz var onun yerine mapperları kullansak modele çevirmek için? Aynı şeyleri tekrar yazmış oluyoruz.

Reply:
mapper test edilmemiş olur o durumda.